### PR TITLE
Added db worker

### PR DESCRIPTION
**Pull request by toshiks**

### CODE REVIEW

File: digestbot/core/DBEngine/utils/createdb.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    """
+    Create tables for current connection
+    :param connection: connection to database
+    :param logger: logger for logging
+    :return: status execution: 0 - Fail, 1 - Success
+    """
+    tables = {"Category": create_category_table(), "Message": create_message_table()}
+
+    big_query = "\n".join(tables.values())
+    logger.info(f"Will be crated new tables: {', '.join(tables.keys())}")
```
Comment (maybe-hello-world) [inline digestbot/core/DBEngine/utils/createdb.py]:
```suggestion
    logger.info(f"Will be created new tables: {', '.join(tables.keys())}")
```

File: digestbot/core/DBModels/Message/Message.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+
+@dataclass
+class Message:
+    id: int
+    username: str
+    text: str
+    date: datetime
```
Comment (maybe-hello-world) [inline digestbot/core/DBModels/Message/Message.py]:
```suggestion
    date: Decimal
```
I have similar dataclass now, we should deduplicate it after merge.
What about putting all dataclasses to separate module?

File: digestbot/core/DBEngine/PostgreSQLEngine.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    def __setting_logger(self):
+        """
+        Setting handlers and level of logging for self.logger
+        """
+
+        self.logger = logging.getLogger("DBEngine")
+        self.logger.setLevel(logging.INFO)
+
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
```
Comment (maybe-hello-world) [inline digestbot/core/DBEngine/PostgreSQLEngine.py]:
```suggestion
        handler.setLevel(logging.DEBUG)  # TODO: change after dockerization
```
As stdout is primary log stream in docker I think it's better to either send level info+ messages or (better) make it configurable later

File: digestbot/core/DBModels/Message/Message.py (hunk 2 above)
Comment (toshiks) [reply digestbot/core/DBModels/Message/Message.py]: If you commit your changes earlier than me - I can pull your changes into my branch. 
But dataclasses - it's about data from out project. Isn't it?

**Pull request merged**